recipes/samarium/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2022 Jai Bellare
+# See <https://opensource.org/licenses/MIT/> or LICENSE.md
+# Project homepage: <https://github.com/strangeQuark1041/samarium>
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.43.0"
+
+
+class SamariumConan(ConanFile):
+    name = "samarium"
+    version = "1.0.0"
```

Comment:
```suggestion
```

Do not include the version, so we can re-use this same recipe for any future release.